stackdriver/lib/stackdriver.rb

```diff
@@ -17,9 +17,9 @@
 gem "google-cloud-logging"
 gem "google-cloud-monitoring"
 
-require "google/cloud/error_reporting/v1beta1"
 require "google/cloud/logging"
-require "google/cloud/monitoring/v3"
+
+require "google/cloud/error_reporting/middleware" if defined? ::Rack
```

Comment:
If we're loading middleware.rb only when ::Rack module is defined, then does middleware.rb still need to [require "rack"](https://github.com/blowmage/gcloud-ruby/blob/611c0bdd0d1b4f9e2487ebfb23c33d847376ae08/google-cloud-error_reporting/lib/google/cloud/error_reporting/middleware.rb#L18-L19) explicitly?